### ISSUE TITLE
Enabling gpu_stats causes crash (SIGABRT) on Intel UHD 770 on Nobara Linux

### ISSUE THREAD

**Issue by MininuxDev** [flightlessmango/MangoHud#1111]
When enabling the gpu_stats option, the ran application displays for a fraction of second and then crashes with the following output:
```
$ mangohud glxgears

[2023-09-05 18:36:41.513] [MANGOHUD] [info] [gl_renderer.cpp:422] GL version: 4.6 
Running synchronized to the vertical refresh.  The framerate should be
approximately the same as the monitor refresh rate.
terminate called after throwing an instance of 'nlohmann::detail::parse_error'
  what():  [json.exception.parse_error.101] parse error at line 59, column 2: syntax error while parsing object - unexpected end of input; expected '}'
Aborted (core dumped)
```

I have installed intel_gpu_tools and ran `sudo setcap cap_perfmon=+ep /usr/bin/intel_gpu_top` to get to this result. Without that, I get  `Failed to initialize PMU! (Permission denied)` and the GPU info field is not displayed at all, but i think that is expected, right ?

- Linux Distribution: Nobara linux 38
- MangoHud version: v0.6.9-1-116-g41f923b+
- GPU: Intel UHD 770
- CPU: Intel i9-13900K
- Motherboard: MSI Tomahawk Z790

**Comment by flightlessmango** [comment]
Can you paste the output of `intel_gpu_top -J` please?

**Comment by MininuxDev** [comment]
Here's what it looks like:

```
{
	"period": {
		"duration": 13.044787,
		"unit": "ms"
	},
	"frequency": {
		"requested": 1226.543599,
		"actual": 1226.543599,
		"unit": "MHz"
	},
	"interrupts": {
		"count": 536.612825,
		"unit": "irq/s"
	},
	"rc6": {
		"value": 0.000000,
		"unit": "%"
	},
	"power": {
		"GPU": 9.825674,
		"Package": 67.829905,
		"unit": "W"
	},
	"engines": {
		"Render/3D/0": {
			"busy": 99.698983,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Blitter/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Video/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Video/1": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"VideoEnhance/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		}
	},
	"clients": {

	}
}
{
	"period": {
		"duration": 1012.981761,
		"unit": "ms"
	},
	"frequency": {
		"requested": 1653.534214,
		"actual": 1635.764891,
		"unit": "MHz"
	},
	"interrupts": {
		"count": 771.978361,
		"unit": "irq/s"
	},
	"rc6": {
		"value": 0.000000,
		"unit": "%"
	},
	"power": {
		"GPU": 9.364938,
		"Package": 54.700415,
		"unit": "W"
	},
	"engines": {
		"Render/3D/0": {
			"busy": 99.747508,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Blitter/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Video/0": {
			"busy": 2.367449,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Video/1": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"VideoEnhance/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		}
	},
	"clients": {

	}
}
{
	"period": {
		"duration": 1013.210104,
		"unit": "ms"
	},
	"frequency": {
		"requested": 1644.278905,
		"actual": 1628.487511,
		"unit": "MHz"
	},
	"interrupts": {
		"count": 746.143369,
		"unit": "irq/s"
	},
	"rc6": {
		"value": 0.000000,
		"unit": "%"
	},
	"power": {
		"GPU": 9.341804,
		"Package": 53.527093,
		"unit": "W"
	},
	"engines": {
		"Render/3D/0": {
			"busy": 99.772505,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Blitter/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Video/0": {
			"busy": 2.316560,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Video/1": {
			"busy": 0.000961,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"VideoEnhance/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		}
	},
	"clients": {

	}
}
{
	"period": {
		"duration": 1013.354932,
		"unit": "ms"
	},
	"frequency": {
		"requested": 1652.925295,
		"actual": 1635.162516,
		"unit": "MHz"
	},
	"interrupts": {
		"count": 645.380981,
		"unit": "irq/s"
	},
	"rc6": {
		"value": 0.000000,
		"unit": "%"
	},
	"power": {
		"GPU": 9.247834,
		"Package": 54.279134,
		"unit": "W"
	},
	"engines": {
		"Render/3D/0": {
			"busy": 99.794049,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Blitter/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Video/0": {
			"busy": 2.339861,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Video/1": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"VideoEnhance/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		}
	},
	"clients": {

	}
}
{
	"period": {
		"duration": 1013.129591,
		"unit": "ms"
	},
	"frequency": {
		"requested": 1652.305899,
		"actual": 1632.565088,
		"unit": "MHz"
	},
	"interrupts": {
		"count": 594.198418,
		"unit": "irq/s"
	},
	"rc6": {
		"value": 0.000000,
		"unit": "%"
	},
	"power": {
		"GPU": 9.430683,
		"Package": 54.029808,
		"unit": "W"
	},
	"engines": {
		"Render/3D/0": {
			"busy": 99.838772,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Blitter/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Video/0": {
			"busy": 2.365073,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Video/1": {
			"busy": 0.000956,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"VideoEnhance/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		}
	},
	"clients": {

	}
}
{
	"period": {
		"duration": 1013.101657,
		"unit": "ms"
	},
	"frequency": {
		"requested": 1645.441984,
		"actual": 1631.623035,
		"unit": "MHz"
	},
	"interrupts": {
		"count": 551.770887,
		"unit": "irq/s"
	},
	"rc6": {
		"value": 0.000000,
		"unit": "%"
	},
	"power": {
		"GPU": 9.429317,
		"Package": 54.224385,
		"unit": "W"
	},
	"engines": {
		"Render/3D/0": {
			"busy": 99.833809,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Blitter/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Video/0": {
			"busy": 2.212024,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"Video/1": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		},
		"VideoEnhance/0": {
			"busy": 0.000000,
			"sema": 0.000000,
			"wait": 0.000000,
			"unit": "%"
		}
	},
	"clients": {

	}

```

**Comment by zmike** [comment]
I'm seeing this too. Would be great to have it fixed.

**Comment by flightlessmango** [comment]
This should be fixed in 9411963
